doc/modules/ensemble.rst

```diff
@@ -603,7 +603,21 @@ fitted model.
 
 ::
 
-  >>> _ = est.set_params(n_estimators=200, warm_start=True)  # set warm_start and new nr of trees
+  >>> import numpy as np
+  >>> from sklearn.metrics import mean_squared_error
+  >>> from sklearn.datasets import make_friedman1
+  >>> from sklearn.ensemble import GradientBoostingRegressor
+
+  >>> X, y = make_friedman1(n_samples=1200, random_state=0, noise=1.0)
+  >>> X_train, X_test = X[:200], X[200:]
+  >>> y_train, y_test = y[:200], y[200:]
+  >>> est = GradientBoostingRegressor(
+  ...     n_estimators=100, learning_rate=0.1, max_depth=1, random_state=0,
+  ...     loss='squared_error'
+  ... ).fit(X_train, y_train)
```

Comment:
Nit: I think it's a clearer to state that we are fitting 100 trees in the initial `fit`

```suggestion
  >>> est = GradientBoostingRegressor(
  ...     n_estimators=100, learning_rate=0.1, max_depth=1, random_state=0,
  ...     loss='squared_error'
  ... )
  >>> est = est.fit(X_train, y_train)  # fit with 100 trees
```

Reply:
Done!